planner/core/logical_plan_builder.go

```diff
@@ -2084,12 +2084,12 @@ func (b *PlanBuilder) buildSelect(ctx context.Context, sel *ast.SelectStmt) (p L
 		p = b.buildTableDual()
 	}
 
-	originalFields := sel.Fields.Fields
 	sel.Fields.Fields, err = b.unfoldWildStar(p, sel.Fields.Fields)
 	if err != nil {
 		return nil, err
 	}
 
+	originalFields := sel.Fields.Fields
```

Comment:
Why not put this in line 2087?

Reply:
because we still need to unfold wildstart...